src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.cs

```diff
@@ -91,7 +91,7 @@ private async Task LoadNativeSplashScreen()
 	}
 
 
-#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP) && !NETSTANDARD2_0
+#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP) && !NET7_0
```

Comment:
I think `NET7_0` will always be true. So, this code is now dead?

Reply:
Yeah I'm not quite sure about this, but what was happening was that WASM was hitting this method impl when it shouldn't be.

Reply:
Yup. Originally Wasm wouldn't hit this because it was `NETSTANDARD2_0`

Thinking about this again, this block of code appears that it should only be true for macOS.

What I would do here is:

- Removing lines 94 to 101 from this file, and move that to a new partial in a new file `ExtendedSplashScreen.macOS.cs`.
- Renaming `ExtendedSplashScreen.Windows.NetStandard.cs` to `ExtendedSplashScreen.Windows.crossruntime.cs`
- Adjust the condition in `ExtendedSplashScreen.Windows.crossruntime.cs` to `﻿#if WINDOWS || WINDOWS_UWP || UNO_REFERENCE_API`
- See if it passes CI.

Reply:
What about for Skia? I believe it should be true for Skia too no?

Reply:
UNO_REFERENCE_API should be true on Skia